src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md

```diff
@@ -32,6 +32,11 @@ You can configure the `OtlpExporter` through `OtlpExporterOptions` properties:
 * `BatchExportProcessorOptions`: Configuration options for the batch exporter.
   Only used if ExportProcessorType is set to Batch.
 
+Some of these can also be set by environment variables following [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/exporter.md). For now we support following variables:
+* `OTEL_EXPORTER_OTLP_ENDPOINT`
+* `OTEL_EXPORTER_OTLP_HEADERS`
+* `OTEL_EXPORTER_OTLP_TIMEOUT`
+
```

Comment:
Can you make the README structure similar to: https://github.com/open-telemetry/opentelemetry-dotnet/blob/5bdebf90039d09dbc5efd5073441b9172f247e58/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md ?

Reply:
Done